Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_disable_system_metrics_and_avoid_logging_them_to_wb.md

---

> title: How can I disable System Metrics and avoid logging them to W&B?

Reviewer: ```suggestion
title: How can I disable logging of system metrics to W&B?
```

toc_hide: true
type: docs
tags:
  - metrics
  - runs
---

> To disable System Metrics:

Reviewer: ```suggestion
To disable logging of [system metrics]({{< relref "/guides/models/app/settings-page/system-metrics.md" >}}), set `_disable_stats` to `True`:
```
I don't find anything about this setting in the public docs. Is that expected?

Author: Yeah we don't mention that, let me check with SDK Team if they're fine with adding this to the docs or if they have any concerns
